process/datadog_checks/process/data/conf.yaml.example

```diff
@@ -69,11 +69,13 @@ instances:
     # ignore_denied_access: true
 
     ## @param thresholds - object - optional
-    ## The threshold paramter is composed of two ranges: critical and warning
+    ## The threshold parameter is composed of two ranges: critical and warning
     ##   * warning: (optional) List of two values: If the number of processes found is below the first value or
-    ##             above the second one, the process check will return WARNING.
+    ##             above the second one, the process check returns WARNING. To make an semi-unbounded interval, 
+    ##             use `.inf` for the upper bound.
```

Comment:
why `.inf`and not `inf`?

Reply:
Just the `yaml` syntax https://yaml.org/refcard.html.
`.inf` is the number infinite
`inf` is the string "inf" I believe